gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright � 2015-2021 the original authors.
+# Copyright � 2015-2021 the original authors.
```

Comment:
`gradlew` still shows up in the patch with changes. You might need to revert it (e.g., using `git checkout master -- gradlew`) from the command line.

Reply:
done.